apps/tenant-manager-cli/src/Infrastructure/Persistence/TenantPersistence.php

```
@@ -0,0 +1,70 @@
+
+        if (isset($data['status'])) {
+            $tenant->setStatus($data['status']);
+        }
+
+        $this->entityManager->persist($tenant);
+        $this->entityManager->flush();
+
+        return $tenant;
+    }
+
+    public function update(string $id, array $data): TenantInterface
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$data'. (undefined)

(UnusedFormalParameter)